.circleci/config.yml

```diff
@@ -279,11 +252,10 @@ jobs:
       - image: circleci/rust:latest
     resource_class: large
     steps:
-      # Need to use nightly for this.
-      - test-setup:
-          rust-version: "nightly"
+      - test-setup
       - install-grcov
-      - run: bash ./automation/emit_coverage_info.sh
+      # Need to use nightly for this.
+      - run: RUSTUP_TOOLCHAIN="nightly" bash ./automation/emit_coverage_info.sh
```

Comment:
I kept this as nightly, because I have this vague belief that coverage stuff "works better" on nightly. I don't really have any justification behind that though, and I expect we could remove this if we wanted, just to simplify things.